scripts/README.md

```diff
@@ -62,6 +66,37 @@ The fleet size should be smaller than or equal to 20, in order to ensure that th
 Please document the displayed bracket-trader addresses. They are required for future withdrawals.
 They can also be retrieved from the created transactions. However, since this is a manual process, it is quite cumbersome to extract them right now.
 
+### Safe Token Distribution
+
+Create your own transferFile, or use our sample [examples/sampleTransferFile.json](examples/sampleTransferFile.json).
+With a fundAccount (aka Gnosis Safe) containg sufficient funds that you own execute:
+
+```sh
+ export PK=<your private key>
+export INFURA_KEY=<your infura key>
+export FUND_ACCOUNT=<your gnosis safe>
+export TRANSFER_FILE=<path to your transfer file>
+```
+
+Alternatively, there is a sample [.env](.env) file that is not tracked by the project where you can paste these values and `source .env`
+
+With all configuration in place, we are ready to run the script.
+
+```sh
+npx truffle exec scripts/airdrop.js --fundAccount $FUND_ACCOUNT --transferFile $TRANSFER_FILE --network rinkeby
```

Comment:
Very nitpicky, consistency with the other examples in the readme:
```suggestion
npx truffle exec scripts/airdrop.js --fundAccount=$FUND_ACCOUNT --transferFile=$TRANSFER_FILE --network=$NETWORK_NAME
```

Reply:
Yes, this is fine, although I am not a fan of the `=` but will accept.